examples/xed-asmparse-main.c

```diff
@@ -30,7 +30,9 @@ END_LEGAL */
 
 #define ASP_MAX_OPERANDS (XED_ENCODER_OPERANDS_MAX+3)
 
-static void process_args(int argc, char** argv, xed_enc_line_parsed_t* v, int* verbose) {
+static void process_args(int argc, char** argv, xed_enc_line_parsed_t* v,
+                         int* verbose, int* quiet) {
```

Comment:
not sure I am psyched about having "verbose" and "quiet" separate variables.  I think I am more comfortable thinking of:

"quiet" as verbose=0, 
"normal" as verbose=1 (default) and
"verbose" as verbose > 1
"debug" as verbose > 2

Would you mind trying that approach instead?

Reply:
Will introduce verbosity levels then